src/library_dyncall.js

```diff
@@ -0,0 +1,67 @@
+mergeInto(LibraryManager.library, {
+  {{{ (function() { global.wbind = function() { return SHRINK_LEVEL == 0 ? 'wbind' : 'wasmTable.get'; }; return null; })(); }}}
+  {{{ (function() { global.getDynCaller = function(sig) { return MINIMAL_RUNTIME ? `dynCalls[${sig}]` : `Module["dynCall_${sig}]`; }; return null; })(); }}}
```

Comment:
This is pretty hard for me to grok.. we have a IIFE embedded in a `{{{ .. }}}`.   Maybe there is some simpler way to express this use `parseTools.js` helpers and normal `#if/#endif` ?

Reply:
This is a way to get around the lack of `#define` support in our preprocessing, since there was never consensus to migrate to using a real preprocessor. Basically this lets have one place where #if is performed, to avoid having to sprinkle the code with several #if #endifs.

Moved to src/parseTools.js.